packages/flutter/test/widgets/image_icon_test.dart

```diff
@@ -33,9 +34,9 @@ void main() {
       ),
     );
 
-    final Image image = tester.widget(find.byType(Image));
-    expect(image, isNotNull);
-    expect(image.color.alpha, equals(128));
+    final Opacity opacity = tester.widget(find.byType(Opacity));
+    expect(opacity, isNotNull);
+    expect(opacity.opacity, equals(0.5));
```

Comment:
i think the previous state of this test represents what we actually want the behaviour to be for optimal performance